### DISCUSSION TITLE
Local notes backup, legacy EN

### DISCUSSION THREAD

**Discussion by vnomad** [vzhd1701/evernote-backup#80] [Q&A]
Things work great from the Windows command line for notes residing on the  EN server.
However, I have >6000 local notes on my drive, and (stupidly) can not figure out how to get them out to .enex.
The built-in export tool chokes and dies after struggling for hours...
I suppose I could export in a bunch of smaller batches, but was hoping for something a bit more ... elegant ;)
Still on legacy EN, ver. 6.25.1.9091

**Comment by vzhd1701** [comment]
Legacy evernote had different database format, evernote-backup works only with the online database at Evernote server.
